File: src/main/java/org/sindeaux/autoconsulta/entityes/Pessoa.java
package org.sindeaux.autoconsulta.entityes;

import jakarta.persistence.*;

import java.time.LocalDate;

@Entity
@Table(name = "TB_PESSOA")
public class Pessoa {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "ID_PESSOA")
    private Long id;

    @Column(name = "TELEFONE_PESSOA")
    private String telefone;

    @Column(name = "IDADE_PESSOA")
    private LocalDate dataNascimento;

    @Column(name = "CPF_PESSOA")
    private String cpf;

    @Column(name = "NOME_PESSOA")
    private String nome;
}

File: src/main/kotlin/com/sindeaux/autoconsulta/entity/Pessoa.kt

import javax.persistence.*

@Entity
@Table(name = "TB_PESSOA")
open abstract class Pessoa {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "ID_PESSOA")
    open var id: Long? = null

    @Column(name = "TELEFONE_PESSOA")
    open var telefone: String? = null

    @Column(name = "IDADE_PESSOA")
    open var idade: Long? = null

    @Column(name = "CPF_PESSOA")
    open var cpf: String? = null

    @Column(name = "NOME_PESSOA")
    open var nome: String? = null
}